### DISCUSSION TITLE
embedded svg not displaying on mobile browser

### DISCUSSION THREAD

**Discussion by tcdk** [JulianPrieber/littlelink-custom#314] [Issues]
Unfortunetly the inline svg's used on "Page Icon" links doesn't work well on mobile browsers based on chrome (I've tried chrome and DuckDuckGo). Here's screenshot from DuckDuckGo: 

![image](https://user-images.githubusercontent.com/557209/214644858-3491757f-26d1-4153-975b-7975a4d1db66.png)

it's from my site https://twomasc.dk 

I've added a twitter link in the page to demonstrate that the icon works fine, when it's a regular img file reference.

I haven't had a chance to test it, but a adding a `xmlns:xlink="http://www.w3.org/1999/xlink"` type may fix it from what I've been finding... https://stackoverflow.com/questions/41507360/svg-inlined-in-html-not-displayed-correctly-by-chrome

But I would rather suggest switching to use a file reference.. like the other links

**Comment by JulianPrieber** [comment]
I can't reproduce this on any device using your provided link, regardless of the browser.

**Comment by lastsamurai26** [comment]
I have tried it on different devices with Chrome, Safari and Edge. 
I was not able to recreate this either. 

Can you check your Dev console ? maybe there is an error which can help us ?

**Comment by JulianPrieber** [comment]
I'm gonna move this one to discussions. Please let us know if there are any updates.

**Comment by tcdk** [comment]
I figured it out! The embedded svg's are failing when the site is http://twomasc.dk , but are displayed in the ssl version at https://twomasc.dk (i guess they are deemed a security risk). This is both mobile and desktop chrome. 

I can of course fix that with a .htaccess redirect, but maybe an option to redirect if the sites loads un-encrypted could be nice
